client/modules/workspace/src/AppsSubmissionForm/AppsSubmissionForm.tsx

```diff
@@ -272,6 +272,16 @@ export const AppsSubmissionForm: React.FC = () => {
     }
   }, [queueValue, setValue]);
 
+  // TODO: DES-2916: Use Zod's superRefine feature instead of manually updating schema and tracking schema changes.
```

Comment:
I did try superRefine, and was not successfully, it was not working or not invoked. This is a bit more involved, hence a follow-up issue.